src/local/watcher.js

```diff
@@ -338,13 +340,14 @@ class LocalWatcher {
           for (const doc of docs.reverse()) {
             if (this.paths.indexOf(doc.path) !== -1 || doc.trashed) {
               continue
+            } else if (doc.docType === 'file') {
+              this.onUnlinkFile(doc.path)
             } else {
-              log.info({path: doc.path}, 'deleted while client was stopped')
-              await this.prep.trashDocAsync(SIDE, doc)
+              this.onUnlinkDir(doc.path)
             }
           }
           delete this.paths
-          callback()
+          setTimeout(callback, 3000)
```

Comment:
I assume having the usual unlinked file/dir timeouts + waiting 3 seconds after initial scan prevents the "onready" race condition?